src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -69,8 +69,10 @@
 
 import static com.minecolonies.api.util.constant.CitizenConstants.*;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of "static" imports in this class from 6 to the maximum allowed 4. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3030)